chart/values.yaml

```diff
@@ -721,6 +721,14 @@ scheduler:
     failureThreshold: 5
     periodSeconds: 60
     command: ~
+
+  # Wait for at most 10 minutes (60*10s) for the scheduler container to startup.
+  # livenessProbe kicks in after the startup
+  startupProbe:
+    failureThreshold: 60
```

Comment:
I thin the defaults are pretty long. If there is an error, the user in default configuraiton will have to waut for 10 minutes before they find out something is wrong. 

Minute as a default should be quite enough IMHO

Reply:
Makes sense. Let me make the changes for that.